### PR TITLE
Update offline maps section for Collect v2024.2

### PR DESCRIPTION
**Pull request by lognaturel**
This page is linked from Collect so it's a common entry point into docs. For this reason, I invested some time to try to make it useful to a broad range of audiences.

I also tried to describe special cases we've considered and make recommendations on how to use the functionality based on some forum conversations.

@alyblenkin and @grzesiek2010 you may want to give this a quick read to see whether I forgot anything.

### CODE REVIEW

File: docs/collect-offline-maps.rst [hunk 1]
```diff
@@ -1,67 +1,122 @@
...
+
+If all devices have access to Google Drive, you can share the MBTiles files from Google Drive. When devices are online, they will be able to select Google Drive from the file picker when they import layers to Collect. This will download and import the files directly. Depending on the devices that you use, you may be able to use a similar process for OneDrive, Dropbox, and other file sharing services.
+
+If all of the devices using Collect will be in the same location, consider sharing the file(s) device-to-device with `Quick Share <https://support.google.com/android/answer/9286773?hl=en>`_, especially if they are large. For example, during a training, you can share the file with one trainee and then ask them to share the file(s) on to the next person and so on.
+
+If your Collect users are remote and don't have access to file sharing services, you can send them a direct link to a file or the file itself via WhatsApp, email, etc. They will then save the file to their Downloads and import it from Collect.
+
+If you are setting up multiple devices from a computer, you may want to :ref:`manually place layer files in the correct directory <transferring-offline-tiles-manually>`.
```
Comment (yanokwa) [inline docs/collect-offline-maps.rst:68]: "correct directory" looks a lot like "collect directory". Replace through the doc

"layers directory in Collect" might be better. Then the note that you can do project specific directory gives more detail.

File: docs/collect-offline-maps.rst [hunk 2]
```diff
@@ -1,67 +1,122 @@
...
+
+.. image:: /img/collect-offline-maps/select-offline-layer.png
+  :class: device-screen-vertical
+
+When you tap the :guilabel:`Add layers` button, you will see a file picker. To import a single file, tap on it and you will go back to Collect to finish the import process. If you need to import multiple offline layers, you can select multiple files. This process will vary a bit by Android version and device type but it is usually done by tapping on their file type icon. After selecting multiple files, a select button will appear at the top or bottom right. Tap that button to confirm your selection and return to Collect.
+
+Once you have selected the layer(s) that you want to import, you will see a confirmation screen showing the layer names and options for sharing the layer(s) between projects or not. This setting is only relevant if you use multiple Collect projects. If you do use multiple projects and they use different offline layers, it can be helpful to import a different list in each individual project. Additionally, when layers are imported at the project level, they will be deleted when the project is deleted.
```
Comment (yanokwa) [inline docs/collect-offline-maps.rst:86]: s/If you do use/if you use

s/import a different list/import a different set/

File: docs/collect-offline-maps.rst [hunk 3]
```diff
@@ -1,67 +1,122 @@
...
+
+.. image:: /img/collect-offline-maps/layer-access.png
+  :class: device-screen-vertical
+
+Once you confirm that you want to add the layer(s), you will see the newly added layer(s) in the layer selection list. The list shows the ``name`` specified by each MBTiles file. You can also see an offline layer's filename by tapping on its name to expand. This can be helpful if the filename is more descriptive than the embedded name.
+
+When Collect imports MBTiles files from device storage, the files are copied, not moved. After they have been imported, it is safe to delete the source files from device storage, usually the Downloads folder, to free up space. Alternately, you can keep source files and delete offline layers from Collect when they are no longer needed. To delete offline layer files, tap on the layer name to see an expanded view that includes a delete button.
```
Comment (yanokwa) [inline docs/collect-offline-maps.rst:93]: s/Alternately/Alternatively/ Do this globally.

File: docs/collect-offline-maps.rst [hunk 4]
```diff
@@ -1,67 +1,122 @@
...
 - from :ref:`mapping-settings`
-- by tapping on the button that looks in a stack of layers in a :ref:`geopoint <geopoint-maps>`, :ref:`geotrace <geotrace-widget>`, or :ref:`geoshape <geoshape-widget>` question
+- by tapping on the button that looks in a stack of layers (:fa:`layer-group`) in a :ref:`geopoint <geopoint-maps>`, :ref:`geotrace <geotrace-widget>`, or :ref:`geoshape <geoshape-widget>` question
 
-Both options set the reference layer for all :ref:`geopoint <geopoint-maps>`, :ref:`geotrace <geotrace-widget>`, and :ref:`geoshape <geoshape-widget>` questions. The choices in the Collect layer selection menu will show the name of the tileset (from the `Metadata table in the MBTiles file <https://github.com/mapbox/mbtiles-spec/blob/master/1.3/spec.md#metadata>`_), as well as the path to the file.
+Both options set the reference layer for all :ref:`geopoint <geopoint-maps>`, :ref:`geotrace <geotrace-widget>`, and :ref:`geoshape <geoshape-widget>` questions in the current project. The choices in the Collect layer selection menu show the name of the tileset from the `Metadata table in the MBTiles file <https://github.com/mapbox/mbtiles-spec/blob/master/1.3/spec.md#metadata>`_.
```
Comment (yanokwa) [inline docs/collect-offline-maps.rst:122]: drop " from the `Metadata table in the MBTiles file` "

File: docs/collect-offline-maps.rst (hunk 3 above)
Comment (lognaturel) [reply docs/collect-offline-maps.rst:93]: Yikes!

**Review by yanokwa — APPROVED** [review]

**Comment by lognaturel** [comment]
Thanks, @alyblenkin! 1 and 2 are going to depend on how the MBTiles files are being generated, Android version, etc so I think they're out of scope. I like the idea of experimenting with a gif showing the whole flow! I'll merge this and file a separate issue.

**Pull request merged**